### ISSUE TITLE
[$500] Submit expense:requests with just one category/tag aren’t automatically selected

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#42091]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.73-0
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @JmillsExpensify 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1715521849768599

## Action Performed:
Prerequisite: Create a collect policy, make sure there is only one possible category, same for tags
1. Go to the staging.new.expensify.com and login
2. Submit an expense without selecting category and tag
## Expected Result:
Expenses with just one possible category/tag are automatically selected.
## Actual Result:
Expenses with just one possible category/tag are NOT automatically selected

## Workaround:
Unknown
## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [X] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

![CleanShot 2024-05-12 at 07 52 09](https://github.com/Expensify/App/assets/38435837/86bb17f4-e459-42ed-b73e-3932233d3cbb)

https://github.com/Expensify/App/assets/38435837/ffbcdd06-e685-4468-a3b3-ae831b15c37b

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01007d3005ad5ee0b2</li>
        <li>Upwork Job ID: 1802794860672433988</li>
        <li>Last Price Increase: 2024-07-16</li>
    </ul>
</details>

**Comment by adelekennedy** [comment]
@m-natarajan it seems like this issue still needs to be filled out?

**Comment by adelekennedy** [comment]
no worries - I was sure I was missing something!

**Comment by JmillsExpensify** [comment]
Putting this on hold until we align [in Slack](https://expensify.slack.com/archives/C049HHMV9SM/p1715521849768599).

**Comment by adelekennedy** [comment]
on hold

**Comment by adelekennedy** [comment]
still on hold

**Comment by adelekennedy** [comment]
bumped the slack thread

**Comment by adelekennedy** [comment]
checked in on the thread again - not adding a project just yet as this may have resolved

**Comment by adelekennedy** [comment]
@JmillsExpensify check me on this - after troubleshooting with Jenna you're still seeing the same issue so we should take this [off hold ](https://expensify.slack.com/archives/C049HHMV9SM/p1717738137799179?thread_ts=1715521849.768599&cid=C049HHMV9SM) and make it a collect issue

**Comment by adelekennedy** [comment]
taking this off hold as @JmillsExpensify is still experiencing this!

**Comment by allroundexperts** [comment]
No proposals.

**Comment by allroundexperts** [comment]
Still looking for proposals.

**Comment by adelekennedy** [comment]
Still waiting for proposals

**Comment by adelekennedy** [comment]
updating the price

**Comment by allroundexperts** [comment]
@adelekennedy Can you confirm that the category / tag should be auto selected even if it is not required? Currently, the logic is to select the single category / tag ONLY if a category / tag is a required field.

If we decide on the former, then @eucool's [proposal](https://github.com/Expensify/App/issues/42091#issuecomment-2231833785) would work well.

🎀 👀 🎀 C+ reviewed

**Comment by aldo-expensify** [comment]
> @adelekennedy Can you confirm that the category / tag should be auto selected even if it is not required? Currently, the logic is to select the single category / tag ONLY if a category / tag is a required field.

@JmillsExpensify I think you reported this issue, can you confirm this 🙏 

> bump @aldo-expensify for assignment

Sorry for the delay, I'll wait for @JmillsExpensify to confirm and then assign

**Comment by aldo-expensify** [comment]
DM'd Jason to get a confirmation on the expected results

**Comment by JmillsExpensify** [comment]
Hmm, interesting. I don't think I realized this when I reported the issue. I do agree that we should only select the category if it's a required field.

**Comment by aldo-expensify** [comment]
I tested creating an expense report in a workspace with a single category that requires categorizing, and the category is automatically selected... Seems like there is nothing to do. I'll close this issue.
